gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -178,7 +178,8 @@ private static void configureErrorProneOptions(
 
         errorProneOptions.setEnabled(true);
         errorProneOptions.setDisableWarningsInGeneratedCode(true);
-        errorProneOptions.setExcludedPaths(project.getBuildDir().getAbsolutePath() + "/.*");
+        errorProneOptions.setExcludedPaths(
+                String.format("%s/(build|src/generated.*)/.*", project.getProjectDir().getAbsolutePath()));
```

Comment:
I wonder if this absolute path is gonna affect the build-cache relocatability of these compile tasks??

Reply:
good call, updating it to be relative path